### PR TITLE
Fio request modal ux change

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: src/components/scenes/RequestScene.js [hunk 1]
```diff
@@ -79,11 +80,14 @@ export type State = {
...
   encodedURI: string,
-  minimumPopupModalState: CurrencyMinimumPopupState
+  minimumPopupModalState: CurrencyMinimumPopupState,
+  isFioMode: boolean
 }
 
 export class Request extends Component<Props, State> {
   amounts: ExchangedFlipInputAmounts
+  flipInput: RefObject | null = null
+  inputAccessoryViewID: string = 'cancelHeaderId'
```
Comment (peachbits) [inline src/components/scenes/RequestScene.js:90]: This is a constant and never edited so it should be outside the class

File: src/components/scenes/RequestScene.js [hunk 2]
```diff
@@ -386,11 +417,11 @@ export class Request extends Component<Props, State> {
     if (this.amounts) {
       const native = parseFloat(this.amounts.nativeAmount)
```
Comment (peachbits) [inline src/components/scenes/RequestScene.js:418]: remove const declration and se bns.lte(native, '0') in if

File: src/components/scenes/RequestScene.js [hunk 3]
```diff
@@ -228,6 +233,17 @@ export class Request extends Component<Props, State> {
     this.onCloseXRPMinimumModal()
   }
 
+  onNext = () => {
+    if (this.state.isFioMode) {
+      this.setState({ isFioMode: false })
+      this.fioAddressModal()
+    }
```
[the remark this answers is not in the document]
Comment (andreyvEze) [reply src/components/scenes/RequestScene.js:240]: I think we don't need else here. There was no onNext before, so if isFioMode === false the behavior stays the same as was before

File: src/components/scenes/RequestScene.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreyvEze) [reply src/components/scenes/RequestScene.js]: done

File: src/components/scenes/RequestScene.js (hunk 2 above)
Comment (andreyvEze) [reply src/components/scenes/RequestScene.js]: done

**Pull request merged**